pkg/blockstorage/vmware/conversion.go

```diff
@@ -0,0 +1,37 @@
+package vmware
+
+import (
+	"github.com/vmware/govmomi/vim25/types"
+
+	"github.com/kanisterio/kanister/pkg/blockstorage"
+)
+
+func convertFromObjectToVolume(vso *types.VStorageObject) *blockstorage.Volume {
+	return &blockstorage.Volume{
+		Type:         blockstorage.TypeFCD,
+		ID:           vso.Config.Id.Id,
+		CreationTime: blockstorage.TimeStamp(vso.Config.CreateTime),
+		Size:         vso.Config.CapacityInMB / 1024,
```

Comment:
do we need any nil checks? will config be non nil?

Reply:
They are not pointers, they must be safe.